android-driver/src/main/scala/com/fortysevendeg/mvessel/api/impl/AndroidCursor.scala

```diff
@@ -1,50 +1,118 @@
 package com.fortysevendeg.mvessel.api.impl
 
-import android.database.Cursor
+import android.content.ContentResolver
+import android.database.{CharArrayBuffer, ContentObserver, Cursor, DataSetObserver}
+import android.net.Uri
+import android.os.Bundle
 import com.fortysevendeg.mvessel.api.CursorType.Value
-import com.fortysevendeg.mvessel.api.{CursorType, CursorProxy}
+import com.fortysevendeg.mvessel.api.{CursorProxy, CursorType}
 
-class AndroidCursor(cursor: Cursor) extends CursorProxy {
+class AndroidCursor(val cursor: Cursor)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/mvessel/file/970112756/issues/source?bid=2442814&fileBranchId=2859700#l10)